src/flipNumber.js

```diff
@@ -145,6 +145,8 @@ export default class FlipNumber extends React.Component<Props, State> {
                     color,
                     background,
                     backfaceVisibility: 'hidden',
+                    WebkitBackfaceVisibility: 'hidden',
```

Comment:
shouldn't this be a fix from auto prefix? otherwise, we have to do transform and everything elese

Reply:
i agree